TestProjects/HDRP_RuntimeTests/Assets/Scenes/002-HDMaterials.unity

```diff
@@ -180,6 +180,7 @@ MeshRenderer:
   m_CastShadows: 1
   m_ReceiveShadows: 1
   m_DynamicOccludee: 1
+  m_StaticShadowCaster: 0
```

Comment:
Why 002 test is change here? it should be affected. Can you revert to master version thanks